_docs/getting-started/mac.md

```diff
@@ -6,22 +6,44 @@ The Mac build works easiest with Anaconda. Always pull the latest from github, s
 
 ## Anaconda Install Path
 
-[Anaconda](https://www.continuum.io/downloads). Python 2.7 version is needed for Caffe2, and Anaconda is recommended. Skip this section to find brew/pip install directions if you are not using Anaconda.
+[Anaconda](https://www.continuum.io/downloads). Python 2.7 version is needed for Caffe2, and Anaconda is recommended. Skip this section to find brew/pip install directions if you are not using Anaconda. The following commands will install caffe2 similarly to how `conda install caffe2` would.
```

Comment:
I am confused. following commands is 'conda install caffe2'. Which is of cause similar to itself :)